src/fast_check/mod.rs

```diff
@@ -26,58 +27,61 @@ fn format_diagnostics(diagnostics: &[FastCheckDiagnostic]) -> String {
 
 #[derive(Debug, Clone, thiserror::Error)]
 pub enum FastCheckDiagnostic {
-  #[error("Could not resolve '{name}' referenced from '{referrer}'. This may indicate a bug in Deno. Please open an issue to help us improve if so.")]
+  #[error("could not resolve '{name}' referenced from '{referrer}'")]
   NotFoundReference {
-    range: Range,
+    range: DiagnosticRange,
     name: String,
     referrer: String,
   },
-  #[error("Missing explicit type in the public API.")]
-  MissingExplicitType { range: Range },
-  #[error("Missing explicit return type in the public API.")]
-  MissingExplicitReturnType { range: Range },
-  #[error("Global augmentations such as ambient modules are not supported.")]
-  UnsupportedAmbientModule { range: Range },
-  #[error("The reference '{name}' from '{referrer}' was too complex. Extract out the shared type to a type alias.")]
```

Comment:
Where are the recommendations for this stored? We should probably keep that in deno_graph rather than elsewhere so that someone can more easily get a diagnostic with the recommendation even if they're not using the CLI.

Reply:
Ok - then I'll move hints, descriptions, etc all into here.